test/unit/submission/validation.spec.ts

```diff
@@ -809,7 +809,7 @@ describe('data-validator', () => {
             donorSubmitterId: 'AB1',
             gender: 'Female',
             programId: 'PEME-CA',
-            sampleSubmitterId: 'AM1',
+            sampleSubmitterId: 'AM2',
```

Comment:
changed this `sampleSubitterId` to not conflict with other record, since this test is only checking `gender` mutation